docs/running-clusterfuzzlite/prow.md

```diff
@@ -0,0 +1,238 @@
+same code coverage. To enable this, add the following to your prow job config for periodic jobs.
+
+{% raw %}
+```yaml
+- name: <YOUR-PROWJOB-NAME>
+  labels:
+    preset-dind-enabled: "true"
+  decorate: true
+  extra_refs:
+  - org: <ORG>
+    repo: <REPO>
+    base_ref: <BRANCH>
+  interval: 1h # THIS CAN BE A CRON AS WELL
+  spec:
+    serviceAccountName: <K8s-SA>
+    containers:
+    - image: gcr.io/k8s-testimages/ci_fuzz:<VERSION>
+      command:
+        - runner.sh
+      args:
+        - python3
+        - "/opt/oss-fuzz/infra/cifuzz/cifuzz_combined_entrypoint.py"
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      env:
+      - name: MODE
+        value: prune
+      - name: SANITIZER
+        value: <SANITIZER>
+      - name: CLOUD_BUCKET
+        value: gs://prow-cifuzz-test/
+      - name: CFL_PLATFORM
+        value: prow
+```
+{% endraw %}
+
+
+## Coverage reports
+
+Periodic coverage reports can also be generated using the latest corpus. To enable this, add the following to your prow job config for periodic jobs.
+
+{% raw %}
+```yaml
+- name: <YOUR-PROWJOB-NAME>
+  labels:
+    preset-dind-enabled: "true"
+  decorate: true
+  extra_refs:
+  - org: <ORG>
+    repo: <REPO>
+    base_ref: <BRANCH>
+  interval: 1h # THIS CAN BE A CRON AS WELL
+  spec:
+    serviceAccountName: <K8s-SA>
+    containers:
+    - image: gcr.io/k8s-testimages/ci_fuzz:<VERSION>
+      command:
+        - runner.sh
+      args:
+        - python3
+        - "/opt/oss-fuzz/infra/cifuzz/cifuzz_combined_entrypoint.py"
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      env:
+      - name: MODE
+        value: coverage
+      - name: SANITIZER
+        value: <SANITIZER>
+      - name: CLOUD_BUCKET
+        value: gs://prow-cifuzz-test/
+      - name: CFL_PLATFORM
+        value: prow
+```
+{% endraw %}
+
+We reccommend scheduling coverage reports to run once a day.
+Note that it's better to schedule this to run after pruning has
+completed, but not required.
+
+## Testing it Out
+
+You can test each of the workflows by [Running the Prowjob Locally](https://github.com/kubernetes/test-infra/blob/master/prow/build_test_update.md#running-a-prowjob-locally)
+
+
+## Storage Layout
+
+Artifacts produced by ClusterFuzzLite are stored in `<your-cloud-bucket>`.
+The bucket has the following layout.
+1. Coverage report: <your-cloud-bucket>/coverage/latest/report/linux/index.html
+1. Builds: <your-cloud-bucket>/build/<sanitizer>-<commit>/
+1. Corpora: <your-cloud-bucket>/corpus/<fuzz-target>/
+1. Crashes: <your-cloud-bucket>/crashes/<fuzz-target>/<sanitizer>/<crash-file> and <your-cloud-bucket>/crashes/<fuzz-target>/<sanitizer>/<crash-file>.summary (output from crash).
+
+## Viewing Crashes
+
+You can view your crashes on Deck by clicking the Artifacts button on the top of the page for your prowjob run. Crashes will be located in 
```

Comment:
A good todo for later would be to get a screenshot of this (need to do this for actions and GCB too).

Reply:
You can leave me a todo and I will add it in a new PR